modules/src/stocks.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import json
+
+from templates.text import TextTemplate
+
+
+def process(userInput, entities):
+    output = {}
+    try:
+        stock_name = entities['stocks'][0]['value'].upper()
+
+        r = requests.get('http://www.google.com/finance?q='+stock_name+'&output=json')
+        data = json.loads(r.text[3:])
+        stockprice = data[0]['l']
+
+        returnString = 'The last price of ' + stock_name + ' is: USD '+ stockprice
+        output['input'] = userInput
+        output['output'] = TextTemplate(returnString).get_message()
+        output['success'] = True
+    except:
```

Comment:
Would be a lot better if you could specify the exceptions. This statement will catch a broad variety of exceptions which won't be something that the script would want.

Reply:
I based my code on another module I just assumed we are doing a generic error message for each module. But sure i can make them more specific based on whats happening.